tomviz/python/tomviz/web.py

```diff
@@ -0,0 +1,167 @@
+import os
+import shutil
+import zipfile
+
+from paraview import simple
+from paraview.web.dataset_builder import ImageDataSetBuilder
+from paraview.web.dataset_builder import CompositeDataSetBuilder
+
+
+def web_export(destinationPath, exportType, deltaPhi, deltaTheta):
+    dest = '%s/data' % destinationPath
+    camera = {
+        'type': 'spherical',
+        'phi': range(0, 360, deltaPhi),
+        'theta': range(-deltaTheta, deltaTheta + 1, deltaTheta)
+    }
+
+    # Copy application
+    copy_viewer(destinationPath)
+
+    # Choose export mode:
+    if exportType == 0:
+        export_images(dest, camera)
+
+    if exportType == 1:
+        export_layers(dest, camera)
+
+    # Zip data directory
+    zipData(destinationPath)
+
+# -----------------------------------------------------------------------------
+# Helpers
+# -----------------------------------------------------------------------------
+
+
+def zipData(destinationPath):
+    dstFile = os.path.join(destinationPath, 'data.tomviz')
+    dataDir = os.path.join(destinationPath, 'data')
+    zf = zipfile.ZipFile(dstFile, mode='w')
+    try:
+        for dirName, subdirList, fileList in os.walk(dataDir):
+            for fname in fileList:
+                fullPath = os.path.join(dirName, fname)
+                relPath = 'data/%s' % (os.path.relpath(fullPath, dataDir))
+                zf.write(fullPath, arcname=relPath,
+                         compress_type=zipfile.ZIP_STORED)
+    finally:
+        zf.close()
+
+    shutil.rmtree(dataDir)
+
+
+def get_proxy(id):
+    session = simple.servermanager.ActiveConnection.Session
+    remoteObj = session.GetRemoteObject(int(id))
+    return simple.servermanager._getPyProxy(remoteObj)
+
+
+def copy_viewer(destinationPath):
+    searchPath = os.getcwd()
```

Comment:
So you are suggesting to use `__file__` as it will be better for the long run and just do a work around when running from the build tree?